vignettes/data_classification.Rmd

```diff
@@ -71,26 +72,26 @@ This dataset serves as a translation key between the GICS sector-classification
 plot_table("gics_classification")
 ```
 
-The `{r2dii.data}` package provides a GICS classification dataset `r2dii.data::gics_classification`.
+The `{pacta.loanbook}` package provides a GICS classification dataset `gics_classification`.
 
 ```{r, R.options = list(width = 400)}
-r2dii.data::gics_classification
+gics_classification
 ```
 
 ## ISIC Rev 5 classification dataset
 
 This dataset serves as a translation key between the ISIC Rev 5 sector-classification system and sector scopes used in the 'PACTA' software.
 
-The `{r2dii.data}` package provides a ISIC Rev 5 classification dataset `r2dii.data::isic_classification`.
+The `{pacta.loanbook}` package provides a ISIC Rev 5 classification dataset `isic_classification`.
```

Comment:
```suggestion
```
duplicated sentence. I removed this one following the structure used for GICS above